assembl/static2/js/app/components/common/avatar.jsx

```diff
@@ -10,27 +11,25 @@ import manageErrorAndLoading from './manageErrorAndLoading';
 import { browserHistory } from '../../router';
 import { localAwareLink } from '../../utils/utilityManager';
 
-class ProfileIcon extends React.Component {
-  constructor(props) {
-    super(props);
-    this.setCurrentView = this.setCurrentView.bind(this);
+type Props = {
+  // used by getDerivedStateFromProps
+  location: string, // eslint-disable-line
```

Comment:
out of curiosity, why disabling this line for eslint ?

Reply:
used by getDerivedStateFromProps